pages/cart.js

```diff
@@ -0,0 +1,251 @@
+import { useContext, useEffect, useState } from "react";
+import styled from "styled-components";
+import { toast } from "react-toastify";
+import { FaTimes } from "react-icons/fa";
+
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import Divider from "../styles/UIKit/Divider";
```

Comment:
all components are imported directly from the UIKit foder "....styles/UIKit", you don't need to write /Divider or /Button here.